### ISSUE TITLE
clnrest plugin complains of flask_restx missing

### ISSUE THREAD

**Issue by Sjors** [ElementsProject/lightning#7383]
I'm not currently using this plugin, but would like to as soon as Zeus supports it, since c-lightning-REST is going away.

### Issue and Steps to Reproduce

It shows in the log every time CLN starts, but can also be triggered by loading the plugin.

```
 lightning-cli plugin start /home/lightning/libexec/c-lightning/plugins/clnrest/clnrest
{
   "code": -3,
   "message": "/home/lightning/libexec/c-lightning/plugins/clnrest/clnrest: disabled itself: No module named 'flask_restx'"
}
```

On my Ubuntu 24.04 machine I've done:

```
sudo apt-get install python3-json5 python3-flask python3-gunicorn
```

The system Python version is 3.12.3

I use PyEnv for my `lightning` user, also version 3.12.3

As this user I've done:

```
pip3 install -r plugins/clnrest/requirements.txt
``` 

Which doesn't complain.

### `getinfo` output

```
{
   "id": "...",
   "alias": "...",
   "color": "...",
   "num_peers": ..,
   "num_pending_channels": 0,
   "num_active_channels": ...,
   "num_inactive_channels": 0,
   "address": [
      {
         "type": "ipv4",
         "address": "...",
         "port": 9735
      },

      {
         "type": "torv3",
         "address": "...",
         "port": 9735
      }
   ],
   "binding": [
      {
         "type": "ipv4",
         ...
      }
   ],
   "version": "v24.05",
   "blockheight": 847358,
   "network": "bitcoin",
   "fees_collected_msat": ...,
   "lightning-dir": "/home/lightning/.lightning/bitcoin",
   "our_features": {
      "init": "080000000000000000000000000008aa880a8a59a1",
      "node": "080000000000000000000000000088aa880a8a59a1",
      "channel": "",
      "invoice": "02000002024100"
   }
}
```

**Comment by ShahanaFarooqui** [comment]
@Sjors Thanks for the issue. It was due to a typo on instructions page . It will be fixed with PR #7400 (Commit https://github.com/ElementsProject/lightning/pull/7400/commits/3eda86c2e93b743987c650493a85b1fcc67e17c1).

**Comment by Sjors** [comment]
Doing either `pip3 install --user flask-restx` or `pip3 install flask-restx` doesn't fix the issue for me, starting the plugin still returns the above error: `No module named 'flask_restx'`.

I tried nuking pyenv for this user. But then `pip3 install --user` fails in general with `error: externally-managed-environment`.

I tried `pip3 install --user --break-system-package` instead, and now it launches!
